src/library.js

```diff
@@ -17,6 +18,7 @@ export default function Library(resolver) {
   const require = requirer(resolver);
   Object.defineProperties(this, {
     DOM: {value: DOM, writable: true, enumerable: true},
+    FileAttachment: {value: constant(FileAttachment), writable: true, enumerable: true},
```

Comment:
The basic implementation looks great and I think the `resolve` function is  exactly the right abstraction here. And I think stdlib is the place for this code too so that we can use it in all the places we need to use it.

That being said, I don't think we want to actually _add_ `FileAttachment` to the library here because by itself we don't want notebook code to be calling it as written. Instead, I think we should just export it from the npm module for use and in practice it'll get wrapped in the context of the situation that can provide a meaningful implementation for the `resolve` function.

Reply:
Great idea. I was imagining that there would be cases where we would only be able to access it from the library itself ... but I guess that's not exactly true.

Reply:
> we don't want notebook code to be calling it as written

Compared to all the other entries in the standard library that run as-is without _much_ further abstraction being applied on top of them. Or in other words, `FileAttachment` here isn't the exact class/function I'm getting when I call `FileAttachment` from a notebook.